### DISCUSSION TITLE
Custom values for frequency filter

### DISCUSSION THREAD

**Discussion by anjalijain22** [broadinstitute/seqr#3302] [Q&A] [answered]
Hello,

I had a question regarding the population frequency filter for variant searches -  currently seqr allows users to select an AF value from predefined values (0.1, 0.05, 0.04 .. etc.). I was wondering if it was possible to use a custom values such as 0.0025 for AF (currently not available in the options or through the scroll bar)?

Thank you in advance for your help!

**Comment by hanars** [comment, marked as answer]
This is not supported in seqr. We recommend you use whatever the next highest frequency option is and then simply don't consider  variants if you really think their frequency is too high for your analysis.

**Reply by anjalijain22** [reply]
Thanks for the quick reply Hana!
